src/BaseTokenWrapper.sol

```diff
@@ -117,23 +117,22 @@ abstract contract BaseTokenWrapper is Ownable, IBaseTokenWrapper {
     uint16 referralCode,
     PermitSignature calldata signature
   ) external virtual {
-    if (signature.deadline != 0) {
-      address debtToken = POOL
-        .getReserveData(TOKEN_OUT)
-        .variableDebtTokenAddress;
-      // explicitly left try-catch block blank to protect users from permit griefing
-      try
-        ICreditDelegationToken(debtToken).delegationWithSig(
-          msg.sender,
-          address(this),
-          amount,
-          signature.deadline,
-          signature.v,
-          signature.r,
-          signature.s
-        )
-      {} catch {}
+    if (signature.deadline < block.timestamp) {
```

Comment:
@miguelmtzinf Should I do this, or keep check the same as before, and just check for general reverting when signature deadline is not properly set?

Reply:
I went back because with 0 deadline, the UI can still function. Still curious if you have comment on this: to explicitly revert with message when deadline on signature is invalid